roslibrust_test/src/ros1.rs

```diff
@@ -28,6 +28,7 @@ pub mod actionlib_msgs {
     }
     impl ::roslibrust_codegen::RosMessageType for GoalID {
         const ROS_TYPE_NAME: &'static str = "actionlib_msgs/GoalID";
+        const MD5SUM: &'static str = "302881f31927c1df708a2dbab0e80ee8";
```

Comment:
Will add tests to the `roslibrust_test` crate in order to ensure that the ways these are calculated don't inadvertently change and that they match what's generated for the C++ headers.